<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="videoButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>video</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="imageButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>image</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="cameraButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>camera</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="stopButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="frame_label">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>28</y>
     <width>521</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="table_label">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>33</y>
     <width>501</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
